src/main/resources/pilotResources/localization/eng/PilotMod-Card-Strings.json

```diff
@@ -164,5 +164,13 @@
     "NAME": "Apex Form",
     "DESCRIPTION": "Whenever you play a card, gain !M! pilot:Momentum.",
     "UPGRADE_DESCRIPTION": "Innate. NL Whenever you play a card, gain !M! pilot:Momentum."
+  },
+  "pilot:Pull_the_Pin": {
+    "NAME": "Pull_the_Pin",
+    "DESCRIPTION": "Add 1 Frag to the top of your draw pile."
+  },
+  "pilot:Frag": {
+    "NAME": "Frag",
+    "DESCRIPTION": "Deal !D! to all enemies."
```

Comment:
This should also state that it is Ethereal and Exhausts. Also, it should read 'Deal !D! damage to...'